system/docker/client/Dockerfile

```diff
@@ -0,0 +1,68 @@
+FROM teracy/ubuntu:16.04-dind-latest
```

Comment:
It is nice to have required ubuntu version and dind in one container, however AFAIK there is another quire widespread and somewhat more secure option - use official dind container + whatever container you need (in this case official ubuntu container) and call docker daemon via local network. If I'm not mistaken this is what [testinfra](https://github.com/philpep/testinfra) does.

Reply:
@skhoroshavin My understanding:
- there is no official dind image except for alpine and ubuntu 14 (if i'm not mistaken)
- testinfra doesn't implement docker client but instead uses shell to run oficial client under the hood https://testinfra.readthedocs.io/en/latest/backends.html
Could you explain your option in more details? Thanks